objective-c/src/Ice/Object.mm

```diff
@@ -77,7 +74,16 @@ virtual void __decRef()
 {
 public:
 
-    BlobjectI(ICEBlobject*);
+    BlobjectI(ICEBlobject* servant) : _blobject(servant), _target(servant)
```

Comment:
I know you didn't change this, but why do we need two pointers to the same object here?

Reply:
You're right, this is unnecessary. I fixed it in #1667.